indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -328,10 +344,10 @@
      (when (or (nil? all-revisions-index?) (not all-revisions-index?))
        (info "Reindexing latest collections for provider" provider-id)
        (let [latest-collection-batches (meta-db/find-in-batches
-                                         context
-                                         :collection
-                                         REINDEX_BATCH_SIZE
-                                         {:provider-id provider-id :latest true})]
+                                        context
```

Comment:
Indentations are off. See above comment for link to coding standard. This applies to lots of the lines in the PR. Seems like you autoformatter is not set up properly.